spec/controllers/admin/server_recordings_controller_spec.rb

```diff
@@ -57,4 +72,22 @@
       end
     end
   end
+
+  describe '#resync' do
+    it 'calls the RecordingsSync service correctly' do
+      expect_any_instance_of(RecordingsSync).to receive(:call)
```

Comment:
I think we may want to assert that the service was initialized with the right parameters and then assert that `:call` was called in the test,